dashboard/app/models/lesson_group.rb

```diff
@@ -180,10 +182,23 @@ def seeding_key(seed_context)
   # }
   #
   # @param [Hash] cb_chapter_data - Chapter data to import.
-  def update_from_curriculum_builder(_cb_chapter_data)
+  # @return [Boolean] - Whether any changes to this lesson group were saved.
+  def update_from_curriculum_builder(cb_chapter_data)
     # In the future, only levelbuilder should be added to this list.
     raise unless [:development, :adhoc].include? rack_env
 
-    # puts "TODO: update lesson group #{id} with cb chapter data: #{cb_chapter_data.to_json[0, 50]}..."
+    cb_questions = cb_chapter_data['questions']
+    if cb_questions.present? && big_questions.blank?
+      self.big_questions = cb_questions
+    end
+
+    cb_description = cb_chapter_data['description']
+    if cb_description.present? && description.blank?
+      self.description = cb_description
+    end
```

Comment:
@bethanyaconnor I am dancing around a bit to avoid overwriting any data that may be there already, but I am not sure if that is worthwhile or not. how are you handling existing data in general -- are you just blowing away whatever is there already, or perhaps manually having to clear it out before re-importing each time as you test things out locally?

Reply:
For lessons, because the way levels are attached to them changed, I've been blowing out fields. If it doesn't matter, then I don't change it but otherwise, I've been assuming that by the time we run it, we'll have tested it enough that we'll know exactly what will happen.